src/components/10-atoms/checkbox/index.js

```diff
@@ -154,7 +169,7 @@ class AXACheckbox extends NoShadowDOM {
       <input
         id="${refId}"
         class="a-checkbox__input"
-        type="checkbox"
+        type="${type}"
```

Comment:
@NoahLiechti Why can a checkbox be of any other type then checkbox? Is radio allowed?

Reply:
@LucaMele You are right it doesn't make sense. But then we can the type attribute completely.

Reply:
Ah somebody forgot to delete it. It's no even declared in `index.react.d.js`